src/xrCore/Threading/ThreadUtil.cpp

```diff
@@ -24,9 +24,15 @@ void SetThreadName(ThreadHandle threadHandle, pcstr name)
         DWORD dwFlags;
     };
 
+    auto namePrefix = "X-Ray ";
+    constexpr auto namePrefixSize = sizeof(namePrefix);
```

Comment:
sizeof namePrefix will return pointer size, because namePrefix will be evaluated to `const char*` instead of `char[]`. At least, this is what IntelliSense says.
```suggestion
    constexpr char namePrefix[] = "X-Ray ";
    constexpr auto namePrefixSize = std::size(namePrefix) - 1;
```

Reply:
It was working fine (I had tested it in debugger), but your version looks better. And also `- 1` is not needed here.